### PR TITLE
Added multiple collisions to safety mode.

### PR DESCRIPTION
**Pull request by yunikkk**

### CODE REVIEW

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -87,7 +87,7 @@
             android:textSize="20sp"
             tools:text="50 m" />
 
-        <com.mapbox.vision.examples.DistanceToCarView
+        <com.mapbox.vision.examples.SafetyModeView
             android:id="@+id/distance_to_car_image"
```
Comment (BenDenen) [inline app/src/main/res/layout/activity_main.xml]: May be it is better to change the ID?

File: app/src/main/java/com/mapbox/vision/examples/activity/main/MainActivity.kt [hunk 2]
```diff
@@ -451,10 +463,10 @@ class MainActivity : AppCompatActivity() {
     private fun setLineDetectionMode() {
         soundsPlayer.stop()
         VisionManager.setModelPerformanceConfig(
-                ModelPerformanceConfig.Separate(
-                        detectionPerformance = ModelPerformance.On(ModelPerformanceMode.FIXED, ModelPerformanceRate.LOW),
-                        segmentationPerformance = ModelPerformance.On(ModelPerformanceMode.FIXED, ModelPerformanceRate.HIGH)
-                )
```
Comment (BenDenen) [inline app/src/main/java/com/mapbox/vision/examples/activity/main/MainActivity.kt]: It is ok for now. But I think It would be better to format code as separate commit.

File: app/src/main/res/layout/activity_main.xml (hunk 1 above)
Comment (yunikkk) [reply app/src/main/res/layout/activity_main.xml]: Ok

File: app/src/main/java/com/mapbox/vision/examples/activity/main/MainActivity.kt (hunk 2 above)
Comment (yunikkk) [reply app/src/main/java/com/mapbox/vision/examples/activity/main/MainActivity.kt]: Ok

**Comment by yunikkk** [comment]
@BenDenen PTAL
